synapse/crypto/keyring.py

```diff
@@ -279,6 +282,11 @@ async def process_request(self, verify_request: VerifyJsonRequest) -> None:
 
         key_ids_to_find = set(verify_request.key_ids) - found_keys.keys()
         if key_ids_to_find:
+            # We're still missing some keys. Consult each of our `KeyFetcher` instances
```

Comment:
Is it possible to set / configure which Synapse / worker instance is a `KeyFetcher`?

Reply:
Is this the answer #14805 ?

Reply:
Yep!